### PR TITLE
doc update

### PR DESCRIPTION
**Pull request by sapana05**
# Description
Update CHANGELOG and README

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE
- Docs Pull Request

##### RESOURCE OR DATASOURCE NAME
CHANGELOG and README

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 80% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- doc updated for release 1.3.0

### CODE REVIEW

**Review by RayLiu7 — APPROVED** [review]

**Review by NavisJ — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ limitations under the License.
 
 The Terraform Provider for Dell Technologies (Dell) PowerScale allows Data Center and IT administrators to use Hashicorp Terraform to automate and orchestrate the provisioning and management of Dell PowerScale storage systems.
 
-The Terraform Provider can be used to manage access zone, active directory, cluster, user, user group, file system, smb share, nfs export, snapshot, snapshot schedule, quota, groupnet, subnet, network pool, network settings, smart pool settings, ldap providers, network rule, file pool policy, ntp server, ntp settings, cluster email settings and acl settings.
+The Terraform Provider can be used to manage access zone, active directory, cluster, user, user group, file system, smb share, nfs export, snapshot, snapshot schedule, quota, groupnet, subnet, network pool, network settings, smart pool settings, ldap providers, network rule, file pool policy, ntp server, ntp settings, cluster email settings, acl settings, nfs export settings,role,user mapping rules.
```
Comment (taohe1012) [inline README.md:24]: `nfs export settings,role,user mapping rules.  ` need one space to split `nfs export settings, role, user mapping rules.` And please add `Role Privilege` here

Comment (forrestxia) [reply README.md:24]: @sapana05 please resolve this comment. thanks!

**Review by forrestxia — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sapana05) [reply README.md:24]: added space and role privilege as requested

**Review by taohe1012 — APPROVED** [review]

**Comment by taohe1012** [comment]
hi @sapana05 , there are 2 commits in the PR, please squash to one, thanks

**Review by taohe1012 — APPROVED** [review]

**Comment by forrestxia** [comment]
e2e pass

**Review by forrestxia — APPROVED** [review]

**Comment by forrestxia** [comment]
@sapana05 signed commit is required.

**Comment by sapana05** [comment]
run e2e test

**Comment by sapana05** [comment]
run e2e test

**Comment by forrestxia** [comment]
e2e pass

**Comment by sapana05** [comment]
run e2e test

**Pull request merged**